start_menu.h

```diff
@@ -0,0 +1,109 @@
+#include <iostream>
+#include "utils.h"
+
+namespace menu
```

Comment:
change the namespace to `start_menu` as well

Reply:
done